src/components/nokkelinfo.tsx

```diff
@@ -119,23 +133,21 @@ const Nokkelinfo = () => {
                 </Heading>
                 <BodyLong className="nokkelinfoContainer">
                     <EnkeltInformasjon header="Telefon" value={telefon ? telefon : EMDASH} />
-                    <EnkeltInformasjon header="Antall barn under 21 år" value={barn.length.toString() || "0"} />
+                    <EnkeltInformasjon header="Antall barn under 21 år" value={barn.length.toString() || '0'} />
                     <EnkeltInformasjon header="Veileder" value={hentVeilederTekst(veileder)} />
                     <EnkeltInformasjon header="Oppfølgingsenhet" value={hentOppfolgingsEnhetTekst(oppfolgingsstatus)} />
                     <EnkeltInformasjon header="Registrert av" value={registrertAv ? registrertAv : EMDASH} />
-                    <EnkeltInformasjon
-                        header="Tilrettelagt kommunikasjon"
-                        value={hentTolkTekst(taletolk)}
-                    />
+                    <EnkeltInformasjon header="Tilrettelagt kommunikasjon" value={hentTolkTekst(taletolk)} />
                     <EnkeltInformasjon header="Sivilstand" value={sivilstatus ? sivilstatus : EMDASH} />
                     <EnkeltInformasjon header="Hovedmål" value={mapHovedmalTilTekst(hovedmaal)} />
-                    <EnkeltInformasjon header="Aktive ytelse(r)" value={ytelserVedtakstype ? ytelserVedtakstype : EMDASH} />
+                    <EnkeltInformasjon header="Aktive ytelse(r)" value={aktivVedtak} />
```

Comment:
Tror vi fortsatt må ha nullsjekk her og eventuelt sende inn EMDASH:

```suggestion
                    <EnkeltInformasjon header="Aktive ytelse(r)" value={aktivVedtak ? aktivVedtak : EMDASH} />
```